webapps/api/tests/test_api_configuration.py

```diff
@@ -65,6 +65,9 @@ def test_return(self):
                     if return_metadata['returns'][0] is not None:
                         errors.append('{0} - Would not expect a return type on a destroy method'.format(fun_id))
             if return_metadata['returns'][0] is None and return_metadata['returns'][1] is None:
+                if fun.__doc__ is None:
```

Comment:
Can't this be higher in this function? On line 38?

Reply:
No, the first ones are checking the decorators.